packages/xy-chart/src/series/BoxPlotSeries.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Group } from '@vx/group';
```

Comment:
I've started doing doing absolute imports like `@vx/group/build/Group` to minimize bundle size (see [issue](https://github.com/williaster/data-ui/issues/49)), could we update these?